layouts/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+{{- /* baseURL is a template.URL, so we need to convert it before we compare it.  */ -}}
+{{- if ne (string .Site.BaseURL) "https://pulumi.com/" }}
```

Comment:
I wonder if we should add the following to our `config.toml` under `[params]`:

```toml
[params]
    productionURL = "https://www.pulumi.com/"
    # ...
```

And then change this comparison to use that, e.g. something like:

```
{{- if ne (string .Site.BaseURL) .Site.Params.productionURL }}
```

Alternatively (and perhaps better)...

I sent you and @chrsmith an email pointing to Hugo's Configuration Directory and `hugo --environment` docs: https://gohugo.io/getting-started/configuration/#configuration-directory

If/when we end up leveraging that to have separate config.toml variables for staging vs. production (and therefore are using `hugo --environment staging` and `hugo --environment production` when building for staging vs. production), we could do something like:

```
{{- if ne .Hugo.Environment "production" }}
```

Reply:
Yep, agreed. I read up a bit on environments yesterday and that seemed like the way to go; I figured we'd do that in the shortish term post-cutover, and come back to make an adjustment here accordingly.